src/qtt/instrument_drivers/virtualAwg/virtual_awg.py

```diff
@@ -59,6 +59,9 @@ def __init__(self, awgs=None, settings=None, name='virtual_awg', logger=logging,
         self._latest_sequence_data = {}
         self.enable_debug = False
 
+        self.settings.snapshot(update=True)
+        self.add_parameter('settings_snapshot', get_cmd = self.settings.snapshot, label='Settings snapshot')
```

Comment:
```suggestion
        self.add_parameter('settings_snapshot', get_cmd=self.settings.snapshot, label='Settings snapshot')
```
Some spaces.

Reply:
Is it possible to add a unit test that makes clear why this functionality is added? When is the settings snapshot updated with the get_cmd?

Reply:
@QFer Added a test, and fixed the bug that was exposed.